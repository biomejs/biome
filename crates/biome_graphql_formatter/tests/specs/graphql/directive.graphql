{
  hero @deprecated
}

{
  hero @deprecated(reason: "Deprecated")
}

{
  hero @input(type: String)
}

{
  hero
		@deprecated(reason: "Deprecated")
		@addExternalFields(source: "profiles")
}
