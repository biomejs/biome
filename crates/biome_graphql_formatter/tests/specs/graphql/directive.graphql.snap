---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: graphql/directive.graphql
---
# Input

```graphql
{


	hero


	@deprecated


}

{
	hero


	@deprecated(reason:

	"Deprecated"

	)


}

{

	hero

	@input

	(type:

	String)


}

{
	hero


	@deprecated(

		reason:

		"Deprecated"

	)
	@addExternalFields


	(source:


	"profiles"

	)
}

```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Bracket spacing: true
Quote style: Double Quotes
-----

```graphql
{
	hero @deprecated
}

{
	hero @deprecated(reason: "Deprecated")
}

{
	hero @input(type: String)
}

{
	hero @deprecated(reason: "Deprecated") @addExternalFields(source: "profiles")
}
```
